tests/backends/mysql/tests.py

```diff
@@ -89,3 +91,24 @@ def test_isolation_level_validation(self):
         )
         with self.assertRaisesMessage(ImproperlyConfigured, msg):
             new_connection.cursor()
+
+
+@unittest.skipUnless(connection.vendor == 'mysql', 'MySQL tests')
+class CallProcTest(BaseCallProcTest, unittest.TestCase):
+    procedure_without_params_sql = '''
```

Comment:
Even though Django doesn't have support for procedures in the ORM, it seems like it would be cleaner if these SQL statements could be retrieved from DatabaseOperations or SchemaEditor. This could allow the tests to be a single class in `tests/backends/test_utils.py` as we're testing `CursorWrapper.callproc()` in `db/backends/utils.py`, correct?